src/utils/constants.ts

```diff
@@ -1,7 +1,16 @@
 import { NotificationVariant } from '../types/notification.types';
 
-export const USE_MOCK_DATA = process.env.REACT_APP_USE_MOCK === 'true';
+const hostToUrlString = (hostString: string | undefined): string => {
+  if (hostString) {
```

Comment:
For readability probably prefer something like (perhaps extract the validation check?):

```
    if (hostString === undefined) {. // although !hostString also works, but is possibly harder to read
      throw new Error(…);
    }
    return …
```